src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/RazorCustomLSPMethodNames.cs

```diff
@@ -3,9 +3,16 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Common;
 
-internal static class RazorLanguageServerCustomMessageTargets
+/// <summary>
+/// This lists all of the LSP methods that we support  that are not part of the LSP spec, or LSP++
+/// </summary>
+/// <remarks>
+/// Handlers for these methods live in either the RazorCustomMessageTarget class in this repo for VS,
+/// or in various TypeScript files in https://github.com/dotnet/vscode-csharp for VS Code.
+/// </remarks>
+internal static class RazorCustomLSPMethodNames
```

Comment:
If it were me, I would just name this `CustomMethodNames` or `CustomRazorMethodNames` and let the namespace do the talking. :smile:

Reply:
Also, if you keep the "LSP" acronym in the name, consider changing it to "Lsp", since [that's standard for .NET APIs](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers) (only two-letter acronyms get a pass 😄).

Reply:
No complaints here, thanks for the feedback